### PR TITLE
Major Refactoring

### PR DESCRIPTION
**Pull request by AAAstorga**
I did a bunch of things. Here is what I remember
- redid all of home screen
  -fixed autolayout on home screen
- added cancelled class cell design
- added tab bar text + images
- did some graphql query stuff
- deleted unused code whenever I ran into it
- cleaned up code when I came across it 
- lots of misc things

tl;dr made the home screen functional across all ios devices with improved autolayout

this PR is certified fresh

### CODE REVIEW

File: Fitness/Networking/NetworkManager.swift [hunk 1]
```diff
@@ -31,244 +32,152 @@ struct NetworkManager {
...
+                guard let gymClassData = gymClassData, let imageUrl = URL(string: gymClassData.imageUrl) else { continue }
+                let instructor = gymClassData.instructor
+                let startTime = gymClassData.startTime ?? "12:00:00"
+                let endTime = gymClassData.endTime ?? "12:00:00"
+                let isCancelled = gymClassData.isCancelled
+                let gymId = gymClassData.gymId ?? ""
+                let location = gymClassData.location
+                let classDescription = gymClassData.details.description
+                let className = gymClassData.details.name
+                let start = Date.convertTimeStringToDate(time: startTime)
```
Comment (jmf373) [inline Fitness/Networking/NetworkManager.swift]: I feel like `convertTimeStringToDate` should take in the queried `date` too, or else the day in the start/end times will always be today, even if we're searching for tmr (could make adding stuff to calendar kinda useless)

File: Fitness/graphql/gymClassInstanceQueries.graphql [hunk 2]
```diff
@@ -10,18 +9,20 @@ query AllClassesInstances
...
       endTime
       instructor
       isCancelled
     }
   }
 
-query TodaysClasses($dateTime: DateTime)
-{
-    classes(now: $dateTime) {
+query TodaysClasses($date: Date) {
```
Comment (jmf373) [inline Fitness/graphql/gymClassInstanceQueries.graphql]: Shouldn't the graphql type be `DateTime` and not `Date`?

File: Fitness/graphql/gymClassInstanceQueries.graphql [hunk 3]
```diff
@@ -10,18 +9,20 @@ query AllClassesInstances
         id
         name
         description
-        tags
+          tags {
+              label
+          }
       }
+      imageUrl
```
Comment (jmf373) [inline Fitness/graphql/gymClassInstanceQueries.graphql]: Is backend adding this attribute? I thought they got rid of it with the new backend

File: Fitness/graphql/gymClassInstanceQueries.graphql (hunk 2 above)
Comment (AAAstorga) [reply Fitness/graphql/gymClassInstanceQueries.graphql]: Kenneth moved it to Date

File: Fitness/graphql/gymClassInstanceQueries.graphql (hunk 3 above)
Comment (AAAstorga) [reply Fitness/graphql/gymClassInstanceQueries.graphql]: Yes, the attribute is added

**Pull request merged**